### DISCUSSION TITLE
How to see the figures in the `question.md`

### DISCUSSION THREAD

**Discussion by EdwardWong1994** [j178/leetgo#225] [General]
Hello, is there a way to see the figure in `question.md`??  It is quite inconvinent just showing a url of figure like the following figure:

<img width="652" alt="image" src="https://github.com/j178/leetgo/assets/96438179/8075d022-6bb1-4992-a4b5-5183f52d3cf5">

**Comment by j178** [comment]
You can use any editor that supports rendering markdown files, like vscode.
